github/event_types.go

```diff
@@ -859,7 +859,7 @@ type MergeGroup struct {
 	HeadRef *string `json:"head_ref,omitempty"`
 	// The SHA of the merge group's parent commit.
 	BaseSHA *string `json:"base_sha,omitempty"`
-	// The full ref of the branch the merge group will be merged into.
+	// The full ref of the branch to which the merge group will be merged.
```

Comment:
The heuristic that complained about this tends to actually trip on sentences that end with prepositions, but I've never actually written a rule for that.